edgexfoundry/app-service-configurable
# feat!: updates for common config

BREAKING CHANGE: configuration file changed to remove common config settings

close #513







**If your build fails**  due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-service-configurable/blob/main/.github/CONTRIBUTING.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions
run edgex non-secure stack with Device Virtual
build service from this branch
run locally with each profile (skipping push-to-core) with `-cp` option
Verify each bootstraps w/o error
For those expecting events from MessageBus, verify they are receiving data (ignore errors processing data for exports)

## New Dependency Instructions (If applicable)
